servicetest/agent/test_agent.py

```diff
@@ -0,0 +1,85 @@
+
+# Copyright (C) 2017 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+
+import pytest
+
+import os
+import time
+
+from testframework import Container, SoftwareContainerAgentHandler
+
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+# Provide what the testframework requires #####
+
+# This function is used by the 'agent' fixture to know where the log should be stored
+def logfile_path():
```

Comment:
We don't seem to ignore this logfile for git anywhere. I propose we follow the same "testoutput dir" pattern as most of the other tests do, and put the log file there, and ignore the directory in .gitignore